app/services/collection_change_set_persister.rb

```diff
@@ -18,14 +18,15 @@ def update
     updated = model
     updated = update_identification(updated) if changed?(:source_id) || changed?(:catkey) || changed?(:barcode)
     updated = updated_access(updated) if access_changed?
+    updated = updated_administrative(updated) if changed?(:admin_policy_id)
```

Comment:
This to me seems like a disjoint change from the other members of this change set.  I would think this should be a separate change set.

Reply:
Say more. This follows the same pattern as the ItemChangeSet (which already updates the APO for a DRO).

Reply:
There are two ways we can take this design, one is that a change set is a proxy object and capable of setting all of the properties.  The other way is that a change set represents an operation on an object.  If the latter, then we'd have separate types of change sets for the different operations the app provides (e.g. change the APO of a Collection)

Reply:
I asked the Valkyrie developers (where this pattern originated) and their rule of thumb is 1 change set per form.

Reply:
What does that buy us except a proliferation of change set classes and boilerplate code? Do we create parallel change set persisters as well?